### ISSUE TITLE
Do not return stringified jsonwebtoken error in Boom

### ISSUE THREAD

**Issue by eventhough** [ideaq/hapi-auth-jwt2#39]
Most of the errors I send back via Boom contain a simple message string, not a stringified JSON object. This is the only error I have to process on the front-end where I need to parse the error body to get the message string back.

This is the line I am talking about: [https://github.com/ideaq/hapi-auth-jwt2/blob/master/lib/index.js#L41](https://github.com/ideaq/hapi-auth-jwt2/blob/master/lib/index.js#L41)

I think adding a stringified JSON object only makes error handling more complicated. There are really only two major types of errors that jsonwebtoken returns, either `TokenExpiredError` or `JsonWebTokenError`. My suggestion would be to just retur a simple string for either error:

- "Token expired"

- "Invalid token"

We can decide on the front-end to use a more friendly error message if we want. My argument is that it seems like this is more in line with how Boom was designed to be used. The structure of a Boom error is so simple:

```
{
    "statusCode": 403,
    "error": "Forbidden",
    "message": "try again some time"
}
```

Thoughts?

**Comment by nelsonic** [comment]
@eventhough sorry to only reply now. 
Agree 100%. 
We were trying to return the jsonwebtoken error.
But you're right, a simple string would be better.
Do you have time to submit a Pull Request?

**Comment by eventhough** [comment]
Working on PR but having trouble passing the real_world_example tests because it looks like I need to set up redis?

**Comment by nelsonic** [comment]
Hi @eventhough you don't have to run your own Redis, just use our dev instance on RedisCloud

Are you using mac/linux/unix? In your terminal run:
```sh
export REDISCLOUD_URL=redis://rediscloud:OhEJHaKRrjWvSgna@pub-redis-10689.eu-west-1-2.1.ec2.garantiadata.com:10689
```
Else if Windows: http://superuser.com/questions/284342/what-are-path-and-other-environment-variables-and-how-can-i-set-or-use-them

Thanks! :+1:

**Comment by nelsonic** [comment]
@eventhough you are now ***contributor*** on this module. :+1: 
Thanks for making it *much better* for everyone! :tada:
Latest version published to NPM: **hapi-auth-jwt2@4.3.2** :white_check_mark:
